alcs-frontend/src/app/shared/footer/footer.component.html

```diff
@@ -0,0 +1 @@
+<p>©BC Government 2022</p>
```

Comment:
maybe something like  new Date().getFullYear()

Reply:
Done changed to be dynamic